packages/ckeditor5-ui/src/search/searchinfoview.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/search/searchinfoview
+*/
+
+import View from '../view';
+
+/**
+ * A view displaying an information text related to different states of {@link module:ui/search/text/searchtextview~SearchTextView}.
+ *
+ * @private
```

Comment:
Did you mean `internal`? I don't really know the impact of having `prrivate` on API docs. https://typedoc.org/tags/private/ describes it only in the meaning of `private` in TypeScript (i.e. class memebers).

Reply:
On the other hand, `internal` is supported by Umberto and the class can be omitted from `.d.ts` files (although, I'm not sure if it's enable in this repo's scripts).